File: strings/gessicaaraujo/1581.py
n = int(input())
while(n > 0):
    n -= 1
    l = []
    m = int(input())
    while(m > 0):
        m -= 1
        l.append(input())
    lang = l[0]
    for m in l:
        if m != lang:
            lang = 'ingles'
            break
    print(lang)

File: strings/laelalbuquerque/1581.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var testes = parseInt(lines.shift());
var idioma, atual = ''
var ingles = false

while (testes--) {
    var pessoas = parseInt(lines.shift());
    idioma = lines.shift().substr(0,19).toLowerCase();
    atual = idioma
    pessoas--
    while (pessoas--) {
        if ((idioma !== atual) && ingles === false){
            ingles = true;

        } else {
            atual = idioma
        }
        idioma = lines.shift().substr(0,19).toLowerCase();
        
    }
    if (ingles){
        console.log("ingles");
    }else {
        console.log(atual);
    }
    ingles = false;
}